tests/platform_tests/sfp/conftest.py

```diff
@@ -77,3 +77,35 @@ def stop_xcvrd(duthost):
     if pmon_file_created:
         duthost.shell('rm {}'.format(pmon_daemon_file_path))
     restart_pmon(duthost)
+
+
+class StopXcvrd:
```

Comment:
@longhuan-cisco no need to stop the Xcvrd. Latest changes in Xcvrd ensures that the DOM is not polled if the port is 'admin' down. @mihirpat1 vis

Reply:
As we discussed in the email, Mihir’s PR [#449](https://github.com/sonic-net/sonic-platform-daemons/pull/449) will only make sure CMIS module’s polling is disabled during interface shut stage. Non-CMIS modules will be always treated as CMIS_STATE_READY after insertion, which causes is_port_dom_monitoring_disabled() to be False no matter whether interface is shut or not.
So to cover both CMIS/non-CMIS modules, we would still need to stop_xcvrd during sfp_reset_cmd, and restart_xcvrd after it.

Reply:
@longhuan-cisco can you disable/enable DOM via:-

`config interface transceiver dom PORT_NAME (enable|disable) `

before and after test

Reply:
@prgeor sure, I updated the diff by using `config interface transceiver dom` to disable/enable dom. Please check.